### ISSUE TITLE
error running `oss-rebuild list pypi absl-py`

### ISSUE THREAD

**Issue by patflynn** [google/oss-rebuild#14]
```bash
***@***:~$ oss-rebuild list pypi absl-py
2024/05/29 15:19:42 initializing GCS client: dialing: google: could not find default credentials. See https://cloud.google.com/docs/authentication/external/set-up-adc for more information
````

Should we add gcloud auth to the instructions?  Is there a way to make this work without requiring auth?

**Comment by wbxyz** [comment]
Huh I thought I tested this without auth, but I guess not well enough.

There's [this line](https://github.com/google/oss-rebuild/blob/c649778815ed938e7b24cb4e221f735df73088f9/cmd/oss-rebuild/main.go#L176) that I thought would make it not required but I'll try to reproduce and figure something else out.

**Comment by wbxyz** [comment]
Ahh I didn't add the same line to the `list` command. I think this should be fixed by #11 @patflynn could you try again now?

**Comment by patflynn** [comment]
works. thanks!
